### PR TITLE
[ticket/16829] forumtitle hover text decoration and missing delimiters

### PR DESCRIPTION
**Pull request by Prosk8er**
Fix text decoration on hover for forumtitle and last subject

PHPBB3-16829

Checklist:

- [x] Correct branch: master for new features; 3.3.x for fixes
- [x] Tests pass
- [x] Code follows coding guidelines: [master](https://area51.phpbb.com/docs/dev/master/development/coding_guidelines.html) and [3.3.x](https://area51.phpbb.com/docs/dev/3.3.x/development/coding_guidelines.html)
- [x] Commit follows commit message [format](https://area51.phpbb.com/docs/dev/3.3.x/development/git.html)

Tracker ticket (set the ticket ID to **your ticket ID**):

https://tracker.phpbb.com/browse/PHPBB3-16829

### CODE REVIEW

**Comment by Prosk8er** [comment]
before:
Screenshot 2021-08-03 170842
Screenshot 2021-08-03 170923
Screenshot 2021-08-03 171028

After:
Screenshot 2021-08-03 171113
Screenshot 2021-08-03 171137
Screenshot 2021-08-03 171152

**Comment by hanakin** [comment]
I have no idea what you are actually changing here can you please in-commit the white space and ; fixes. They are outside the scope of this or. Need there own pr. It's just difficult to review with all the clutter.

File: phpBB/styles/prosilver/theme/links.css [hunk 1]
```diff
@@ -90,13 +90,15 @@ a.lastsubject:hover {
 }
 
 .row-item a:hover {
-	text-decoration: none
+	text-decoration: none;
 }
 
+.row-item .forumtitle:hover,
```
Comment (Prosk8er) [inline phpBB/styles/prosilver/theme/links.css:96]: @hanakin this is the part that i'm mainly fixing line 96 to 101

Comment (hanakin) [reply phpBB/styles/prosilver/theme/links.css:96]: Yes but this still goes against our guidelines. We try to keep prs focused on fixing a single thing to allow for easier merging and undoing. This should be split up into separate prs.

Comment (hanakin) [reply phpBB/styles/prosilver/theme/links.css:96]: Do a pr for just that fix, then a new one for the ;

Comment (Prosk8er) [reply phpBB/styles/prosilver/theme/links.css:96]: ok will do i'll just close this one cause that's easier for me and start the new ones

**Pull request closed without merging**